### ISSUE TITLE
a bug when use train-100h and cpc-phone

### ISSUE THREAD

**Issue by liu-x-p** [andi611/Self-Supervised-Speech-Pretraining-and-Representation-Learning#57]
Could there be a little bug in the code?
I got great accuracy when I used train-100h subset to pretrain. I notice when using train-100h subset to pretrain the upstream model, it may load all of the data, but when test with cpc-phone, it use a split of the train-100h subset. The model may have already seen the test data.
I'm not sure if it is right.
Looking forward for you reply.

**Comment by andi611** [comment]
During pre-train, the model only sees the audio part (x).
During supervised phone classification training, the model sees the pair data: audio & phone label (x, y).

There is no problem with the train-clean-100 subset, because the label data (y) is not seen during pre-training.
Plus, this is a popular setup used in many papers (CPC by DeepMind, Modified CPC by Facebook AI).
